projects/igniteui-angular/src/lib/directives/button/button.directive.ts

```diff
@@ -217,6 +254,7 @@ export class IgxButtonDirective extends DisplayDensityBase {
 
 /**
  * @hidden
+ * @internal
```

Comment:
@3phase , according to the latest API docs guidelines, modules can't be `@internal`

Reply:
Thanks for the remark, it's done.